docs/mdbook/default.nix

```diff
@@ -66,11 +66,11 @@ let
         # Make sure this path has a valid info attrset
         if info ? file && info ? description && info ? url then
           "# ${lib.last path}\n\n"
-          + (lib.optionalString (info.description != null) "${info.description}\n\n")
           + (lib.optionalString (info.url != null) "**URL:** [${info.url}](${info.url})\n\n")
           + (lib.optionalString (
             maintainers != [ ]
           ) "**Maintainers:** ${lib.concatStringsSep ", " maintainersNames}\n\n")
+          + (lib.optionalString (info.description != null) "${info.description}\n\n")
```

Comment:
Re: https://github.com/nix-community/nixvim/pull/2062#issuecomment-2305432338

```suggestion
          + lib.optionalString (info.description != null) ''

            ---

            ${info.description}\n\n
          ''
```

1. The parentheses aren't needed here
2. Lets attempt to have some separation

Could also try a header:

```suggestion
          + lib.optionalString (info.description != null) ''

            ## Description

            ${info.description}\n\n
          ''
```

But IDK if "description" is the best heading for generic documentation, even if that _is_ the internal name...

Reply:
I think a divider looks pretty clean 
![image](https://github.com/user-attachments/assets/7032bd85-21ce-41c0-a3db-a327efb37fed)